docs/renewables/wind/method.rst

```diff
@@ -0,0 +1,27 @@
+From Wind Speed to Power Output
+###############################
+Once the U and V components of the wind are converted to a non-directional wind speed
+magnitude, this speed is converted to power using wind turbine power curves. Since real
+wind farms are not currently mapped to a grid network farms, a capacity-weighted average
+wind turbine power curve is created for each state based on the turbine types reported
+in `EIA Form 860`_. The wind turbine curve for each real wind farm is looked up from a
+database of curves (or the *IEC class 2* power curve provided by NREL in the `WIND
+documentation`_) is used for turbines without curves in the database), and scaled from
```

Comment:
There is an unmatched closed parenthesis here, either at `database)` or `WIND documentation_)`.

Reply:
Done